TestModels/Extendable/runtimes/go/simpleextendableresourcesinternaldafnynativeresourcefactory/simpleextendableresourcesinternaldafnynativeresourcefactory.go

```diff
@@ -32,6 +30,6 @@ type Dummy__ struct{}
 
 func DafnyFactory() simpleextendableresourcesinternaldafnytypes.IExtendableResource {
 	extendableResource := ExtendableResource.New_ExtendableResource_()
-	extendableResource.OfName(_dafny.SeqOfString("dafny-default"))
-	return &simpleextendableresources.NativeWrapper{Impl: simpleextendableresources.ToNativeExtendableResource(extendableResource)}
+	extendableResource.OfName(dafny.SeqOfString("dafny-default"))
+	return &simpleextendableresources.NativeWrapper{Impl: simpleextendableresources.ExtendableResource_FromDafny(extendableResource)}
```

Comment:
Why don't I see the corresponding change for `ExtendableResource_FromDafny` since this rename would require method rename in the package as well?

Reply:
Few months ago we changed `ToNative` to `FromDafny` but we did not change this manually written file.